BirdClubManagementSystem/Views/TournamentStandings/Create.cshtml

```diff
@@ -31,10 +31,10 @@
                     <div class="select-box">
                         <select asp-for="Placement">
                             <option value="" hidden>Choose a ranking</option>
-                            <option value="1st">First place</option>
-                            <option value="2nd">Second place</option>
-                            <option value="3rd">Third place</option>
-                            <option value="Participation">Participation</option>
+                            <option value="1St">First Place</option>
```

Comment:
any way to not hard code this? I remember we have class that holds these as static fields.